src/plugins/data_source/common/data_sources/types.ts

```diff
@@ -10,6 +10,7 @@ export interface DataSourceAttributes extends SavedObjectAttributes {
   description?: string;
   endpoint: string;
   dataSourceVersion?: string;
```

Comment:
nit: prolly keeping in line with version which makes sense too.

but might be redundant to put `dataSource` in the field name since it's apart of the DataSource attributes. but i also understand the concept of data sources has different meanings even within this repo so might make sense to specify.

Reply:
yet you got the point, the `dataSource` prefix was added intentionally just to distinguish with other data source terms